cli/tsc/mod.rs

```diff
@@ -575,14 +577,12 @@ fn op_resolve(
     )?
   };
   for specifier in args.specifiers {
-    if let Some(module_name) = specifier.strip_prefix("node:") {
-      if deno_node::is_builtin_node_module(module_name) {
-        // return itself for node: specifiers because during type checking
-        // we resolve to the ambient modules in the @types/node package
-        // rather than deno_std/node
-        resolved.push((specifier, MediaType::Dts.to_string()));
-        continue;
-      }
+    if specifier.starts_with("node:") {
+      resolved.push((
+        MISSING_DEPENDENCY_SPECIFIER.to_string(),
+        MediaType::Dts.to_string(),
+      ));
+      continue;
     }
```

Comment:
Will this cause invalid node specifiers to not surface an error?

Reply:
Still works, those errors aren't surfaced from returning `None` here.